### PR TITLE
Working on #2972 Template Conversion to Twig Format (video)

### PR DESCRIPTION
**Pull request by Shade-**
For #2972. Templates converted:
- [x]  video_dailymotion_embed
- [x]  video_facebook_embed
- [x]  video_liveleak_embed
- [x]  video_metacafe_embed
- [x]  video_myspacetv_embed
- [x]  video_twitch_embed
- [x]  video_veoh_embed
- [x]  video_vimeo_embed
- [x]  video_yahoo_embed
- [x]  video_youtube_embed

### CODE REVIEW

**Comment by euantorano** [comment]
We've got more than a few conflicts here due to the rebase @Shade-. @PaulBender found it easier to simply recreate his PRs, so that might be the way to go.

File: inc/views/base/parser/video.twig [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{% if type == 'dailymotion' %}
```
Comment (euantorano) [inline inc/views/base/parser/video.twig]: I wonder if it's worth splitting these out into partial templates? This seems a bit messy to me personally.

File: inc/class_parser.php [hunk 2]
```diff
@@ -1458,8 +1458,11 @@ function mycode_parse_video($video, $url)
 
 		$id = htmlspecialchars_uni($id);
```
Comment (euantorano) [inline inc/class_parser.php]: This will lead to double escaping as the `id` is also escaped in the template.

Comment (Shade-) [reply inc/class_parser.php]: Thank you for spotting this, will get rid of it.

File: inc/views/base/parser/video.twig (hunk 1 above)
Comment (Shade-) [reply inc/views/base/parser/video.twig]: Having 10+ templates for such a repetitive thing doesn't look clean personally, but I can push a PR if you want :)

Comment (euantorano) [reply inc/views/base/parser/video.twig]: We can leave it as is for now, and if people provide feedback against it, we can always split them out easily at a later date - with twig we don't need PHP changes anymore afterall 😉

**Pull request merged**